src/main/scala/mesosphere/marathon/api/HttpTransferMetricsHandler.scala

```diff
@@ -86,17 +86,17 @@ class HttpTransferMetricsHandler(httpMetrics: HttpTransferMetrics) extends Abstr
     if (!isExcluded(request)) {
       val bytesRead = request.getContentLengthLong
       val bytesWritten = request.getResponse.getContentCount
-      if (bytesRead > 0) // bytesRead can be -1 if no entity provided
+      if (bytesRead > 0) { // bytesRead can be -1 if no entity provided
+        httpMetrics.bytesReadMetric.increment(bytesRead)
```

Comment:
This is the compressed length. Right? Is there any way we can get the uncompressed length instead?

Reply:
Yes, there is a way. It wasn't as obvious but the Jetty HttpInputStream has a counter.

Additionally, we do not support gzipped posts. I looked at the Jetty gzip handler. It's simply not implemented. So I just removed the gzipped metric.

Reply:
Also, for bytes written, it is decompressed length. So the same as what we were tracking before.